src/config.rs

```diff
@@ -321,6 +329,19 @@ impl Context {
                     .set_raw_config(key.as_ref(), value.as_deref())
                     .await?;
             }
+            Config::DebugLogging => {
+                if value == Some("0") || value == Some("") || value.is_none() {
+                    if let Some(webxdc_message_id) = self
+                        .sql
+                        .get_raw_config_u32(Config::DebugLogging.as_ref())
+                        .await?
+                    {
+                        message::delete_msgs(self, &[MsgId::new(webxdc_message_id)]).await?;
+                    }
+                    self.sql.set_raw_config(key.as_ref(), None).await?;
+                    self.debug_logging.store(0, atomic::Ordering::Relaxed);
```

Comment:
I don't really want anything to happen when setting debug logging to 1 because that must only happen after the debug logging webxdc was posted and set on the context anyway. As above, the config should be able to set from off -> on in the ui

Reply:
In any case, this asymmetry is confusing to read and should be changed in one way or the other (always change `context.debug_logging` or never)